hack/lib/etcd.sh

```diff
@@ -40,12 +40,12 @@ kube::etcd::start() {
   # Start etcd
   ETCD_DIR=$(mktemp -d -t test-etcd.XXXXXX)
   kube::log::usage "etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null"
-  etcd -data-dir ${ETCD_DIR} -addr ${host}:${port} >/dev/null 2>/dev/null &
+  etcd -data-dir ${ETCD_DIR} --bind-addr ${host}:${port} >/dev/null 2>/dev/null &
   ETCD_PID=$!
 
   echo "Waiting for etcd to come up."
-  kube::util::wait_for_url "http://${testhost}:${port}/v2/machines" "etcd: " 0.25 80
-  curl -X PUT "http://${testhost}:${port}/v2/keys/_test"
+  kube::util::wait_for_url "http://${host}:${port}/v2/machines" "etcd: " 0.25 80
```

Comment:
This was code I put in before I figured out what was wrong with -addr vs -bind-addr